'use strict';

const chai = require('chai');
const InternalException = require('../../src/Exception/InternalException');

const expect = chai.expect;

describe('InternalException', () => {
    it('Constructor', () => {

    });
});
// const Exception = require('./Exception.test');
// const log = require('../utils/log');
//
// class InternalException extends Exception {
//     constructor(...errorDetails) {
//         super(500, 'An internal error ocurred');
//         log.console(...errorDetails);
//     }
// }
//
// module.exports = InternalException;
